### PR TITLE
fix(gemini): set gemini base image to debian

### PR DESCRIPTION
**Pull request by soyacz**
Currently we use `busybox` for gemini base image. Busybox uses different C standard library implementation and this may lead to some unknown errors.

Switch base image to `minideb` for Gemini so we can exclude this kind of errors from possible culprits of our problems with container segfaults when using Gemini.

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by fruch** [comment]
@soyacz 
please fix one of unittests that expect a specific version of gemini image.

**Comment by soyacz** [comment]
@fruch unit tests/integration fixed

**Comment by fruch** [comment]
> @fruch unit tests/integration fixed

Good, still try to have a short longevity using it, to be on the safe side

File: unit_tests/test_config.py [hunk 1]
```diff
@@ -178,13 +178,13 @@ def test_10_mananger_regression():
     def test_12_scylla_version_ami(self):
         os.environ.pop('SCT_AMI_ID_DB_SCYLLA', None)
         os.environ['SCT_CLUSTER_BACKEND'] = 'aws'
-        os.environ['SCT_SCYLLA_VERSION'] = '4.6.3'
+        os.environ['SCT_SCYLLA_VERSION'] = '5.1.8'
```
Comment (fruch) [inline unit_tests/test_config.py:181]: can you move all the those integration test fixes to it's own commit ? i.e. those are not related to gemini docker image

Comment (soyacz) [reply unit_tests/test_config.py:181]: done

**Comment by soyacz** [comment]
> > @fruch unit tests/integration fixed
> 
> Good, still try to have a short longevity using it, to be on the safe side

I tried yesterday, but issues with raft group0 stopped the test. I'll retry later

**Comment by fruch** [comment]
> > > @fruch unit tests/integration fixed
> > 
> > 
> > Good, still try to have a short longevity using it, to be on the safe side
> 
> I tried yesterday, but issues with raft group0 stopped the test. I'll retry later

update us when you retry it again

**Comment by soyacz** [comment]
gemini test rerun: https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/lukasz/job/gemini-3h-with-nemesis-test/13/

**Review by fruch — APPROVED** [review]
LGTM

**Comment by fruch** [comment]
For now, lets not backport, until we can see if it helped with anything
